app.ts

```diff
@@ -54,75 +53,25 @@ const createError = (
   return new IntentFlow.HandlerError(requestId, errorCode, msg);
 };
 
-/** Get custom data stored in devices from a specified lookup key. */
-const getCustomDataByKey = (
+/** Get Home Assistant info stored in device custom data. */
+const getHassCustomData = (
```

Comment:
We can only ever sync a single instance, so we just find the first device that has Home Assistant info stored in it's custom data.